### PR TITLE
Add versions for Unified Service Tagging env vars

### PR DESCRIPTION
**Pull request by andrewardito**
### What does this PR do?
Add minimum tracer versions to setup tables for environment variables. PHP to be modified when 0.47 is released, and Ruby should be changed in gem doc.

### Preview link
https://docs-staging.datadoghq.com/andrew.ardito/unified-service-tagging-tracer-versions/

### CODE REVIEW

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ With these three tags you can:
 
 - Unified service tagging requires setup of the [Datadog Agent][2].
 
-- Unified service tagging requires knowledge of configuring tags. If you are unsure of how to configure tags, read the [Getting Started with Tagging][3] and [Assigning Tags][4] documentation before proceeding to configuration.
+- Unified service tagging requires a tracer version that supports the [reserved tags][1]. More information can be found per language in the [setup insructions][3].
```
Comment (jdgumz) [inline content/en/getting_started/tagging/unified_service_tagging.md:30]: nit: supports new configuration of the reserved tags.

File: content/en/tracing/setup/go.md [hunk 2]
```diff
@@ -55,10 +55,10 @@ Datadog has a series of pluggable packages which provide out-of-the-box support
 ## Configuration
 
 The Go tracer supports additional environment variables and functions for configuration.
-See all available options in the [configuration documentation][72].
+See all available options in the [configuration documentation][11].
 
 We highly recommend using `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` to set `env`, `service`, and `version` for your services.
-Check out the [Unified Service Tagging][73] documentation for recommendations on how to configure these environment variables.
+Check out the [Unified Service Tagging][12] documentation for recommendations on how to configure these environment variables. These variables are available for version 1.24.0+ of the Go tracer.
```
Comment (jdgumz) [inline content/en/tracing/setup/go.md:61]: nit: `version` or `versions`? We have `versions` in the .NET table above.

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 1 above)
Comment (jdgumz) [reply content/en/getting_started/tagging/unified_service_tagging.md:30]: also a typo in `instructions`

**Comment by andrewardito** [comment]
> I think we need to cover the versions that support the new config for Ruby as well.
> 
> Otherwise this looks good!

Ruby is single sourced from https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md. so it has to be updated there.

**Pull request merged**